ordata/dae/manifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<dae_root>./man1.dae</dae_root>
```

Comment:
I am nervous about ROS detecting `manifest.xml` as a `rosbuild` package. Could you add `CATKIN_IGNORE `and `ROS_NOBUILD` files in the `ordata` directory to guarantee that this does not happen?